.env.example

```
@@ -106,6 +134,20 @@ MINIO_API_PORT=8009
 # DEFAULT: 8010
 MINIO_BROWSER_PORT=8010
 
+# Public port to the webdav server.
+# NOTE: Ignored if `webdav` is not used.
+# DEFAULT: 8020
+WEBDAV_PUBLIC_PORT=8020
+
+# Credentials of the user in the webdav server.
+# NOTE: Ignored if webdav is not used.
+WEBDAV_USERNAME=qfc_user
+WEBDAV_PASSWORD=qfc_pwd
```

Makes it explicit where it is used just by looking at the values:
```suggestion
WEBDAV_USERNAME=qfc_webdav_user
WEBDAV_PASSWORD=qfc_webdav_pwd
```

---

Also, add documentation before each envvar, do not group them like this. With a default value.